### PR TITLE
[Bug Fix]show partitions add absence column for filter columns check

### PR DESCRIPTION
**Pull request by zhuxt2015**
## What type of PR is this：
- [x] bug
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/analysis/ShowPartitionsStmt.java [hunk 1]
```diff
@@ -44,6 +47,18 @@ public class ShowPartitionsStmt extends ShowStmt {
     public static final String FILTER_REPLICATION_NUM = "ReplicationNum";
     public static final String FILTER_LAST_CONSISTENCY_CHECK_TIME = "LastConsistencyCheckTime";
 
+    public static final Set<String> FILTER_COLUMNS;
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/analysis/ShowPartitionsStmt.java:50]: How about using ImmutableSet

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 10 / 10 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowStmtAnalyzer.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/ShowPartitionsStmt.java|8|8|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 8 / 8 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/ShowStmtAnalyzer.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/analysis/ShowPartitionsStmt.java|6|6|100.00%|[]|

**Review by imay — APPROVED** [review]

**Comment by imay** [comment]
run starrocks_admit_test

**Pull request merged**